Loop/Managers/CGMManager.swift

```diff
@@ -6,18 +6,9 @@
 //
 
 import LoopKit
-import CGMBLEKit
-import G4ShareSpy
-import ShareClient
 
 
-let allCGMManagers: [CGMManager.Type] = [
-    G6CGMManager.self,
-    G5CGMManager.self,
-    G4CGMManager.self,
-    ShareClientManager.self,
-]
-
+let allCGMManagers = Bundle.linkedCGMManagers.sorted(by: { $0.localizedTitle < $1.localizedTitle })
```

Comment:
If we sort by localized title, will that give us the order we want? I assume we'd want the more recent CGMs (eg. G6) before the older ones.